tests/test_queuing.py

```diff
@@ -326,67 +257,74 @@ def test_append_qubit_observables(self):
 
     def test_append_tensor_ops(self):
         """Test that ops which are used as inputs to `Tensor`
-        are successfully added to the queue, but no `Tensor` object is."""
+        are successfully added to the queue, as well as the `Tensor` object."""
 
-        with qml._queuing.AnnotatedQueue() as q:
+        with AnnotatedQueue() as q:
             A = qml.PauliZ(0)
             B = qml.PauliY(1)
             tensor_op = qml.operation.Tensor(A, B)
-        assert q.queue == [A, B]
+        assert q.queue == [A, B, tensor_op]
         assert tensor_op.obs == [A, B]
-        assert all(not isinstance(op, qml.operation.Tensor) for op in q.queue)
 
     def test_append_tensor_ops_overloaded(self):
         """Test that Tensor ops created using `@`
-        are successfully added to the queue, but no `Tensor` object is."""
+        are successfully added to the queue, as well as the `Tensor` object."""
 
-        with qml._queuing.AnnotatedQueue() as q:
+        with AnnotatedQueue() as q:
             A = qml.PauliZ(0)
             B = qml.PauliY(1)
             tensor_op = A @ B
-        assert q.queue == [A, B]
+        assert q.queue == [A, B, tensor_op]
         assert tensor_op.obs == [A, B]
-        assert all(not isinstance(op, qml.operation.Tensor) for op in q.queue)
 
     def test_get_info(self):
         """Test that get_info correctly returns an annotation"""
         A = qml.RZ(0.5, wires=1)
 
-        with qml._queuing.AnnotatedQueue() as q:
+        with AnnotatedQueue() as q:
             q.append(A, inv=True)
 
-        assert q.get_info(A) == {"inv": True}
+        assert q._get_info(A) == {"inv": True}
```

Comment:
Why was this changed?

Reply:
When we completed the queuing refactor last year in tape mode, we renamed this method in the new core (but not the old core).